### PR TITLE
docs: add sci-o documentation

### PR DESCRIPTION
**Pull request by jmcook1186**
### Types of changes

- [x] Enhancement (project structure, spelling, grammar, formatting)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### A description of the changes proposed in the Pull Request

Updates documentation for`sci-o` model and adds example impl that invokes `sci-o`.

### CODE REVIEW

File: examples/impls/sci-o.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+      pipeline:
+        - sci-e
+        - sci-o
+      config:
+        sci-e:
+      observations:
+        - timestamp: 2023-08-06T00:00
+          duration: 3600
+          e-cpu: 0.001
+          grid-ci: 800
```
Comment (jmcook1186) [inline examples/impls/sci-o.yml:22]:
```suggestion
          grid-ci: 800

```

Comment (jmcook1186) [inline examples/impls/sci-o.yml:22]:
```suggestion
          grid-ci: 800
          
```

**Review by gnanakeethan — APPROVED** [review]

File: docs/implementations/sci-o.md [hunk 2]
```diff
@@ -1,30 +1,29 @@
...
 
-## Scope
+## Calculation
 
-To calculate the operational emissions O for a software application, use the following:
+To calculate the operational emissions `o` for a software application, use the following:
 
-O = (E * I)
+```
+o = (energy * grid-ci)
```
Comment (gnanakeethan) [inline docs/implementations/sci-o.md:10]: The SCI specification mentions O , E, I as the variables. I think it will be better to keep them as is.

Comment (jmcook1186) [reply docs/implementations/sci-o.md:10]: Fair enough, but this is consistent with the model implementation, and I find longer names easier to understand compared to remembering abbreviations. Keep both?

Comment (gnanakeethan) [reply docs/implementations/sci-o.md:10]: Keep both is better I think. SCI specification mentions or so, Therefore we can say equivalent terms in the implementation is this.

**Pull request merged**